### ISSUE TITLE
Wrong top and botton plane.  

### ISSUE THREAD

**Issue by dinosauria123** [Ondsel-Development/FreeCAD#34]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

I draw three over wrapped circles on sketcher and extrude them as a solid at the same time, It shows strange rectangle planes at top and bottom of the objects.
![Screenshot from 2024-03-14 09-22-44](https://github.com/Ondsel-Development/FreeCAD/assets/3067670/e8fd1781-4a92-40a3-98ce-0d5f6b1e0994)
![Screenshot from 2024-03-14 09-23-39](https://github.com/Ondsel-Development/FreeCAD/assets/3067670/5f834b10-075f-4d10-8104-5b39d725e8d3)
[test.FCStd.zip](https://github.com/Ondsel-Development/FreeCAD/files/14595165/test.FCStd.zip)

### Full version info

```shell
OS: Ubuntu 23.10 (ubuntu:GNOME/ubuntu-wayland)
Word size of Ondsel: 64-bit
Version: 2024.2.0dev.36204 (Git) AppImage
Build type: Release
Branch: flavor
Hash: cfe8ba1084699057a1367965d315c118e4aabeeb
Python 3.10.13, Qt 5.15.8, Coin 4.0.0, Vtk 9.2.6, OCC 7.6.3
Locale: Japanese/Japan (ja_JP)
Installed mods: 
  * Ondsel-Lens 2024.1.22.01
  * InventorLoader 1.4.0
  * sheetmetal 0.4.2
  * OpenDark 2023.12.17
```
```


### Subproject(s) affected?

None

### Anything else?

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by prokoudine** [comment]
This isn't an Ondsel specific thing, it can be replicated in upstream FreeCAD as well. I think the program is expecting a kind of a sketch that can be extruded/padded into a solid body, but you are not giving it the right geometry. E.g. it needs to concentric circles to create a hollow cylinder. However, when you create offsets for the circles and trim intersection for them, it pads just fine. You'll also need a separate body for the cylinder that doesn't intersect other cylinders.

[2024-04-07-2.zip](https://github.com/Ondsel-Development/FreeCAD/files/14898720/2024-04-07-2.zip)

**Comment by prokoudine** [comment]
BTW, Onshape has a problem interpreting the same case as well and does it differently, by summing two intersecting circles into one shape:

![image](https://github.com/Ondsel-Development/FreeCAD/assets/57467/fc21b50e-ea35-477a-ba2e-051bfa63f4f6)

Unless, of course, this is how you were expecting Ondsel ES to behave :) In which case, please tell us what you expected to happen apart from not seeing the two planes.

**Comment by dinosauria123** [comment]
Thank you for your comments. I hope this will be "Fixed".

**Comment by dinosauria123** [comment]
Thank for implementing "Pad: Result has multiple solids: that is not currently supported." in Ondsel ES 2.0.

Stay tuned.

**Comment by adrianinsaval** [comment]
"result has multiple solids" message exists for years now. This was either a bug in freecad's sketch profile validation that was fixed between then an now or this was an upstream opencascade bug that was fixed between 7.6.3 (used in previous release) and 7.7.2 (used now)
